src/lib_gui/qt/graphics/QtGraphicsView.cpp

```diff
@@ -166,6 +165,8 @@ QtGraphicsView::QtGraphicsView(GraphFocusHandler* focusHandler, QWidget* parent)
 	m_legendButton->setObjectName(QStringLiteral("legend_button"));
 	m_legendButton->setToolTip(QStringLiteral("show legend"));
 	connect(m_legendButton, &QPushButton::clicked, this, &QtGraphicsView::legendClicked);
+
+	m_zoomFactor = ApplicationSettings::getInstance()->getGraphZoomLevel();
```

Comment:
You could keep setting `m_zoomFactor` in the initialization list of the constructor.